README.md

```diff
@@ -46,12 +46,128 @@ The `main` and `db` packages contain integration tests that require a successful
 ### No-scan domains
 In case of complaints or abuse, we may not want to continually scan some domains. You can set the environment variable `DOMAIN_BLACKLIST` to point to a file with a list of newline-separated domains. Attempting to scan those domains from the public-facing website will result in error codes.
 
-## API
+## Scan API
 
+Our API objects can look a bit complicated! There's lots of information contained in a TLS scan.
 To request a scan:
 ```
 POST /api/scan
+  { "domain": "example.com" }
+```
+
+SLet's break down exactly what each part of this giant nested response means. All API responses, not just scans, are wrapped in a JSON, like:
+```
+{
+    status_code: 200,
+    message: "",
+    response: <reponse data>
```

Comment:
I fear you're getting ahead of yourself and writing the French version of the API documentation already! The S from "SLet's" above would be better off here in the word "reponse". :-)